pkg/reconciler/mtbroker/resources/channel_test.go

```diff
@@ -34,103 +28,3 @@ func TestBrokerChannelName(t *testing.T) {
 		t.Errorf("expected %q, actual %q", expected, actual)
 	}
 }
-
-func TestNewChannel(t *testing.T) {
-	testCases := map[string]struct {
-		channelTemplate messagingv1.ChannelTemplateSpec
-		expectError     bool
-	}{
-		"InMemoryChannel": {
-			channelTemplate: messagingv1.ChannelTemplateSpec{
-				TypeMeta: v1.TypeMeta{
-					APIVersion: "messaging.knative.dev/v1alpha1",
-					Kind:       "InMemoryChannel",
-				},
-			},
-		},
-		"KafkaChannel": {
```

Comment:
It seems okay to remove some of these tests, especially the KafkaChannel test, as it should be the responsibility of eventing-kafka. But there seem to be other things being thrown out by removing this test entirely, e.g., the BrokerChannelName check. Any reason for this?

Reply:
The reason is that the tested function here doesn't exist anymore. There is still a test for `BrokerChannelName`

Reply:
I see. And the OwnerReferences test done by `assertSoleOwner`?

Reply:
`assertSoleOwner` was an assert executed by the deleted test, and note that now the `OwnerReferences` is set by the caller of `NewPhysicalChannel`